modules/color-me/commands/color-me.js

```diff
@@ -39,19 +49,24 @@ module.exports.subcommands = {
                         {
                             name: interaction.options.getString('name'),
                             color: roleColor,
+                            icon: roleIcon,
                             reason: localize('color-me', 'edit-log-reason', {
                                 user: interaction.user.username
                             })
                         }
                     );
-                    await interaction.editReply(await embedType(moduleStrings['updated'], {}, {ephemeral: true}));
+                    if (iconW) {
+                        await interaction.editReply(embedType(moduleStrings['updated'], {}, {ephemeral: true}));
```

Comment:
```suggestion
                        await interaction.editReply(embedType(moduleStrings['updated']));
``
Not needed here, as the original reply is already ephemeral

Reply:
This is still the case for most calls of editReply

Reply:
i think i fixed it now